src/Tasks/BootstrapperUtil/BootstrapperBuilder.cs

```diff
@@ -2092,7 +2092,7 @@ private bool AddVerificationInformation(XmlNode packageFileNode, string fileSour
                     }
 
                     // If the public key in the file doesn't match the public key on disk, issue a build warning
-                    if (publicKey == null || !publicKey.ToLowerInvariant().Equals(publicKeyAttribute.Value.ToLowerInvariant()))
+                    if (publicKey == null || !publicKey.Equals(publicKeyAttribute.Value, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
nit:
We normally use StringComparison.OrdinalIgnoreCase. It doesn't really matter in this case (or the other one), but I'd slightly prefer it.